web/src/app/services/data-store/data-store.service.ts

```diff
@@ -179,7 +179,11 @@ export class DataStoreService {
       .collection(SURVEYS_COLLECTION_NAME, ref =>
         ref
           .where(s.generalAccess, '==', GeneralAccess.UNLISTED)
-          .where(s.ownerId, '==', userId)
+          .where(
```

Comment:
The conditions in Firestore are applied incrementally, meaning this query will include results where the survey is UNLISTED _AND_ the user is the survey organizer.

Reply:
yes, there are three different queries. one for public, one for restricted and one for unlisted. this is is for the unlisted one and only to show them in the survey list.

Reply:
Ah ok, can you maybe rename the variable and/or add a comment to make it more obvious that we're looking for "unlisted surveys which the current user manages?"

Reply:
done. i've chosen to rename the variables. :)